configure.ac

```diff
@@ -141,6 +142,15 @@ case "$with_tcmalloc_alignment" in
   *)
        AC_MSG_WARN([${with_tcmalloc_alignment} bytes not supported, using default tcmalloc allocation alignment.])
 esac
+AC_ARG_ENABLE([libc-override],
```

Comment:
Let's keep the option, the macro, the help, etc phrase in terms of an enable rather than a disable.
It's okay that it's an enable that defaults to "yes".
It's easier when we can avoid the double negatives like ifndef DISABLE_FOO || DISABLE_FOO!=1,
which makes me a little cross-eyed here.

Reply:
They have a couple of other parameters written as "disables". I thought this made most sense as the default behavior is enable and therefore you can disable it. I know it looks really weird in autoconf as they are written as "enables". But --enable-no-libs-override is double negation and also very confusing.